src/main/scala/fi/oph/koski/validation/MaksuttomuusValidation.scala

```diff
@@ -16,26 +17,56 @@ object MaksuttomuusValidation {
                                         oppijanHetu: Option[String],
                                         opiskeluoikeusRepository: CompositeOpiskeluoikeusRepository)
                                        (implicit user: KoskiSpecificSession): HttpStatus = {
-    val suoritusVaatiiMaksuttomuusTiedon = oppivelvollisuudenSuorittamiseenKelpaavaMuuKuinPeruskoulunOpiskeluoikeus(opiskeluoikeus)
+    val aikaraja = LocalDate.of(2021, 1, 1)
```

Comment:
Tämä päivämääräraja olisi mielstäni lukea Valppaan rajapäivät-palvelusta, koska siellä se on konfiguroitavissa. Samoin  myös oppijanIkäOikeuttaaMaksuttomuuden -rajapäivä.

Jos todetaan, että näitä ei tarvitse konfiguroida, niin kovakoodausta parempi olisi joka tapauksessa kovakoodata nämä sinne rajapäivätService:een, ettei tule kylvettyä useampaan paikkaan koodia.